### PR TITLE
[DOC] add examples of query with requests for adapters

### PR DESCRIPTION
**Pull request by bojiang**
## Description

As title.

## Motivation and Context

https://github.com/bentoml/BentoML/issues/1201

## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature and improvements (non-breaking change which adds/improves functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Code Refactoring (internal change which is not user facing)
- [x] Documentation
- [ ] Test, CI, or build

## Component(s) if applicable
- [ ] BentoService (service definition, dependency management, API input/output adapters)
- [ ] Model Artifact (model serialization, multi-framework support)
- [ ] Model Server (mico-batching, dockerisation, logging, OpenAPI, instruments)
- [ ] YataiService gRPC server (model registry, cloud deployment automation)
- [ ] YataiService web server (nodejs HTTP server and web UI)
- [ ] Internal (BentoML's own configuration, logging, utility, exception handling)
- [ ] BentoML CLI

## Checklist:

- [x] My code follows the bentoml code style, both `./dev/format.sh` and
  `./dev/lint.sh` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [ ] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: bentoml/adapters/annotated_image_input.py [hunk 1]
```diff
@@ -113,6 +113,23 @@ def predict(
...
 
+    OR by python requests:
+
+    .. code-block:: python
+
+        import requests
+
+        with open("test.jpg", "rb") as f:
+            image_bytes = f.read()
+        with open("anno.json", "rb") as f:
```
Comment (parano) [inline bentoml/adapters/annotated_image_input.py:124]: give the user an alternative to use some in-memory data as the JSON annotation data?

e.g.
```
annotations = { "age": 10, "bar": "foo" }
anno_bytes = json.dumps(annotations).encode()
```

File: bentoml/adapters/file_input.py [hunk 2]
```diff
@@ -101,6 +101,20 @@ def predict(self, file_stream):
...
         </form>
 
+    OR by python requests:
+
+    .. code-block:: python
+
+        import requests
+
+        with open("test.jpg", "rb") as f:
+            image_bytes = f.read()
```
Comment (parano) [inline bentoml/adapters/file_input.py:111]: Similarly let's give the user another example to turn a PIL image object into a request?

File: bentoml/adapters/annotated_image_input.py (hunk 1 above)
Comment (pietermarsman) [reply bentoml/adapters/annotated_image_input.py:124]: That is definitely helpful! I think in many cases the annotation part of the request is created at runtime and not read from file.

**Review by pietermarsman — APPROVED** [review]
This is very helpful!

**Pull request merged**